aspose/src/test/java/nl/nn/adapterframework/extensions/aspose/pipe/PdfPipeTest.java

```diff
@@ -123,9 +125,12 @@ public void expectSuccessfullConversion(String pipeName, String fileToConvert, S
 			log.debug("converted relative path ["+expectedFile+"] to absolute file ["+expectedFilePath+"]");
 
 			PDFUtil pdfUtil = new PDFUtil();
+			pdfUtil.setCompareMode(CompareMode.VISUAL_MODE);
```

Comment:
I'd like to also set `allowedDeviation` here, just to be more explicit in the tests and not rely on defaults like we did with the compare-mode.